modules/oaeservice/manifests/puppet.pp

```diff
@@ -7,4 +7,15 @@
   class { 'puppetdb::master::config':
     puppetdb_server => 'puppet',
   }
-}
\ No newline at end of file
+
+  # clean puppet report logs
+  # the report dir is not in puppet-hilary anywhere so extracted direct from puppet.conf
+  cron { 'clean-puppet-reports':
+    ensure  => present,
+    command => "find $(awk -F= '/^reportdir/ { print $2 }' /etc/puppet/puppet.conf) -type f -iname \*.yaml -ctime +30 -delete",
```

Comment:
Can this directory be defaulted to `/var/lib/puppet/reports` somehow? The QA pupper server doesn't seem to configure a reportdir